diesel_cli/src/cli.rs

```diff
@@ -57,6 +57,9 @@ pub fn build_cli() -> App<'static, 'static> {
         ).subcommand(SubCommand::with_name("drop")
             .about("Drops the database specified in your DATABASE_URL.")
             .setting(AppSettings::Hidden)
+        ).subcommand(SubCommand::with_name("touch")
```

Comment:
But `setup` also runs the migrations, which I don't want - the state left behind by this by the example can't be handled by those in `diesel_tests` (which have a different migration history, but overlapping table name `posts`).